README.md

```diff
@@ -0,0 +1,25 @@
+# Sawtooth PoET
+
+Sawtooth PoET is a consensus engine for Hyperledger Sawtooth based on a
+solution to Byzantine Generals Problem that utilizes
+a “trusted execution environment(TEE)”.
+
+The PoET specification is available in
+[sawtooth documentation]( https://sawtooth.hyperledger.org/docs/core/nightly/master/architecture/poet.html)
+
+Currently, Sawtooth PoET implements the non-sgx simulator mode written as a python application.
```

Comment:
How about: "The Sawtooth PoET consensus engine implements a CFT (Crash Fault Tolerant) variant of PoET which does not use a TEE. A future version of the consensus engine which also implements full BFT (Byzantine Fault Tolerant) features using an SGX enclave is in development."

Reply:
Updated